plugins/com.google.cloud.tools.eclipse.usagetracker/src/com/google/cloud/tools/eclipse/usagetracker/AnalyticsPingManager.java

```diff
@@ -259,7 +263,7 @@ public void showOptInDialog(final Shell parentShell) {
       display.syncExec(new Runnable() {
         @Override
         public void run() {
-          new OptInDialog(findShell(parentShell)).open();
+          new OptInDialog(findShell(parentShell), dialogOpenStatus).open();
```

Comment:
There was a bit of a discussion around this, and we decided that implicit closure of a dialog is considered opting out. (So, a user will see the opt-in dialog only once.)